specification/components/schemas/eRS-Appointment.yaml

```diff
@@ -46,18 +45,79 @@ properties:
       - cancelled
       - noshow
     example: proposed
+  reason:
```

Comment:
we only accept 'booked' or 'proposed' here - I wonder whether we need an explanation